atmsimulator/account.py

```diff
@@ -1,55 +1,122 @@
+    print('Your savings are: ',globals.account['users'][0]['accounts'][0]['balance'] ,' Rs.')
+    time.sleep(3)
+    reset()
 
 
 def customer_account_cash_withdrawal():
     # Implement functionality to deduct money from respective account
     # and print withdrawal message to the customer
-    pass
+    if globals.config['account']['withdrawal']!=1:
+           disabled_service_message()
+    else:
+        question = [  inquirer.List('selection',
+                        message='Choose account to withdraw',
+                        choices=['Savings', 'Current', 'Cancel']
+                        )]
+        selection = inquirer.prompt(question)['selection']
+
+        if selection == 'Savings':
+            withdraw = int(input('Enter amount in Rs. to be withdrawn: '))
+            if globals.account['users'][0]['accounts'][0]['balance'] < withdraw:
+                print('Your account has insufficient balance.')
+                reset()              
+            globals.account['users'][0]['accounts'][0]['balance']-=withdraw
+            print('Withdraw successful. Your savings balance is: ', globals.account['users'][0]['accounts'][0]['balance']) 
+
+        elif selection == 'Current':
+            withdraw = int(input('Enter amount in Rs. to be withdrawn: '))
+            if globals.account['users'][0]['accounts'][1]['balance'] < withdraw:
+                print('Your account has insufficient balance.')
+                reset()               
+            globals.account['users'][0]['accounts'][1]['balance']-=withdraw
+            print('Withdraw successful. Your current balance is: ', globals.account['users'][0]['accounts'][1]['balance'])
+
+        else:
+            clear()
+            unsuccessful_transaction_message()
 
 
 def customer_account_cheque_deposit():
     # Implement functionality to add cheque funds into respective
     # account and print confirmation message
-    pass
+    question = [  inquirer.List('selection',
+                      message='Choose account to deposit cheque',
+                      choices=['Savings', 'Current', 'Cancel']
+                      )]
+    selection = inquirer.prompt(question)['selection']
+
+
+    if selection == 'Savings':
+        deposit = int(input('Enter amount in Rs. to be deposited: '))
+        globals.account['users'][0]['accounts'][0]['balance']+=deposit
+        print('Deposit successful. Your savings balance is: ',globals.account['users'][0]['accounts'][0]['balance'])  
+    elif selection == 'Current':
+        deposit = int(input('Enter amount in Rs. to be deposited: '))
+        globals.account['users'][0]['accounts'][1]['balance']+=deposit
+        print('Deposit successful. Your current balance is: ',globals.account['users'][0]['accounts'][1]['balance'])  
+    else:
+        clear()
+        unsuccessful_transaction_message()
 
 
 def customer_account_mini_statement():
```

Comment:
I believe @nkilm had written an implementation of mini statement, did you check with him before you made changes?